tests/test_catalog.py

```diff
@@ -463,3 +463,30 @@ def test_catalog_json(path: str, glue_database: str, glue_table: str) -> None:
     )
     partitions_values = wr.catalog.get_partitions(database=glue_database, table=glue_table)
     assert len(partitions_values) == 2
+
+
+def test_catalog_legacy_delta(path: str, glue_database: str, glue_table: str) -> None:
+    # Create Legacy Delta table
```

Comment:
Could we please add a pytest exception like [here](https://github.com/awslabs/aws-data-wrangler/blob/7db106c6ed6a6804fded88ed4c22b0645ac3524f/tests/test_catalog.py#L434) to test for a missing `_symlink_format_manifest` in the path?